pages/common/ssh.md

```diff
@@ -25,8 +25,12 @@
 
 - SSH tunneling: Forward a specific port (localhost:9999 to slashdot.org:80):
 
-`ssh -L {{9999}}:slashdot.org:80 {{username}}@{{remote_host}}`
+`ssh -L {{9999}}:slashdot.org:80 -N {{username}}@{{remote_host}}`
```

Comment:
Ah, good shout. This makes it consume fewer server resources. Perhaps we could add `-T` here too (disable pseudo-terminal allocation)?

Reply:
Wasn't aware of that but makes sens